### ISSUE TITLE
Compliation issue

### ISSUE THREAD

**Issue by srikanth430** [RedisLabs/JReJSON#5]
When trying to install the JReJSON,

mvn clean install -Dmaven.test.skip=true

It throws an error,
[ERROR] C:\Softwares\JReJSON-master\src\main\java\com\redislabs\modules\rejson\JReJSON.java:[133,20] error: void cannot be dereferenced
did anyone get past this issue? If yes please let me know how.

**Comment by davzhou** [comment]
Looks like jedis was updated recently

**Comment by daniel-cohen** [comment]
This is the 1st time I'm using this library and I'm just trying to build it, so I am not sure if it's working yet  (if it does I'll fork it and submit a PR), but it builds with the following changes:
```
diff --git a/src/main/java/com/redislabs/modules/rejson/JReJSON.java b/src/main/java/com/redislabs/modules/rejson/JReJSON.java
index a2767b1..df487cd 100644
--- a/src/main/java/com/redislabs/modules/rejson/JReJSON.java
+++ b/src/main/java/com/redislabs/modules/rejson/JReJSON.java
@@ -29,6 +29,8 @@
 package com.redislabs.modules.rejson;
 
 import com.google.gson.Gson;
+
+import redis.clients.jedis.Client;
 import redis.clients.jedis.Jedis;
 import redis.clients.jedis.commands.ProtocolCommand;
 import redis.clients.util.SafeEncoder;
@@ -128,9 +130,11 @@ public class JReJSON {
         args.add(SafeEncoder.encode(key));
         args.add(SafeEncoder.encode(getSingleOptionalPath(path).toString()));
 
-        Long rep = conn.getClient()
-                    .sendCommand(Command.DEL, args.toArray(new byte[args.size()][]))
-                    .getIntegerReply();
+        //TODO:DJC
+        Client client = conn.getClient(); 
+        client.sendCommand(Command.DEL, args.toArray(new byte[args.size()][]));
+        Long rep =  client.getIntegerReply();
+        
         conn.close();
 
         return rep;
@@ -152,9 +156,12 @@ public class JReJSON {
             args.add(SafeEncoder.encode(p.toString()));
         }
 
-        String rep = conn.getClient()
-                .sendCommand(Command.GET, args.toArray(new byte[args.size()][]))
-                .getBulkReply();
+        
+        //TODO: DJC
+        Client client = conn.getClient();
+        client.sendCommand(Command.GET, args.toArray(new byte[args.size()][]));
+        String rep = client.getBulkReply();
+        
         conn.close();
 
         assertReplyNotError(rep);
@@ -180,9 +187,11 @@ public class JReJSON {
             args.add(flag.getRaw());
         }
 
-        String status = conn.getClient()
-                .sendCommand(Command.SET, args.toArray(new byte[args.size()][]))
-                .getStatusCodeReply();
+        //TODO: DJC
+        Client client = conn.getClient();
+        client.sendCommand(Command.SET, args.toArray(new byte[args.size()][]));
+        String status =  client.getStatusCodeReply();
+        
         conn.close();
 
         assertReplyOK(status);
@@ -213,9 +222,11 @@ public class JReJSON {
         args.add(SafeEncoder.encode(key));
         args.add(SafeEncoder.encode(getSingleOptionalPath(path).toString()));
 
-        String rep = conn.getClient()
-                .sendCommand(Command.TYPE, args.toArray(new byte[args.size()][]))
-                .getBulkReply();
+        //TODO:DJC
+        Client client = conn.getClient(); 
+        client.sendCommand(Command.TYPE, args.toArray(new byte[args.size()][]));
+        String rep = client.getBulkReply();
+        
         conn.close();
 
         assertReplyNotError(rep);

```

**Comment by gabrielcc2** [comment]
Hi,

Adding to this issue:  Some months ago the SafeEncoder was moved into Jedis.Utils. (xetorthio/jedis#1689)

Given this I got some errors when building (which anyone would probably get if they install the current Jedis 3.0.0-SNAPSHOT into their local repository). Changing the import path to: import redis.clients.jedis.util.SafeEncoder; helps.

In addition the changes mentioned by Daniel Cohen here are needed to get to use what's currently in the master branch.

I hope that pointing this out is useful.

Cheers,

Gabriel
